bond_test.go

```diff
@@ -38,6 +45,11 @@ func (a *Account) CollectionName() string {
 	return DB.Account.Name()
 }
 
+func (a Account) AfterCreate() {
```

Comment:
good point about the only way to reference the DB is through a global.. its reasonable to assume the session is passed here